### PR TITLE
Bump promscale helm chart to 13.0.0

### LINKED ISSUE #509: Update how we set Promscale image/tag
Creating this issue to remind me if [timescale/promscale#1520](https://github.com/timescale/promscale/pull/1525) is merged, we will need to refactor how we set the container image and tag in `values.yaml` for Promscale.

This will need to be updated only when the PR is approved and the Promscale Helm chart version is updated.

### PR DESCRIPTION
**Pull request by nhudson**
#### What this PR does / why we need it

Update Promscale Helm chart to the latest version `13.0.0` and update relevant documentation

#### Which issue this PR fixes

- fixes #509 

#### Checklist

- [x] Chart Version bumped
- [x] [CLA signed](https://cla-assistant.io/timescale/tobs)

### CODE REVIEW

**Comment by nhudson** [comment]
Do not merge this until after #546

File: chart/Chart.yaml
Comment (onprem) [inline chart/Chart.yaml:18]: Since the `image` format change is not backwards compatible, I think we should bump the major version.

**Review by onprem — APPROVED** [review]
LGTM!

**Pull request merged**
